package.json

```diff
@@ -24,7 +24,6 @@
   "files": [
     "CHANGELOG.md",
     "index.d.ts",
-    "index.js",
```

Comment:
@cjbarth hi, Is this change correct?

I already read this: [configuring-npm/package-json#files](https://docs.npmjs.com/cli/v9/configuring-npm/package-json#files)
>Certain files are always included, regardless of settings: The file in the "main" field

But with `npm pack` i get this:

![image](https://github.com/node-saml/xml-crypto/assets/109294935/8fd39e71-cebc-4d4e-a938-2af90128c513)

`index.js` is missing, i'm doing something wrong?? or is `main` failing because it is `./index.js`

Reply:
if you remove `./` it works
https://github.com/node-saml/xml-crypto/blob/6bcbaa6f602aef67f6f5a291b3760ce00b67d997/package.json#L22-L23
Maybe here too
https://github.com/node-saml/xml-crypto/blob/6bcbaa6f602aef67f6f5a291b3760ce00b67d997/package.json#L30

Reply:
Good find. I was just about to do a release. Please make a PR for this and we'll get it in :) I've been jumping around too much and missed this. Thanks for keeping an eye on this!